### DISCUSSION TITLE
빌드잇에서 작업한 맵을 유니티에 불러왔는데, 캐릭터가 물체를 통과하여 계속해서 추락합니다.

### DISCUSSION THREAD

**Discussion by LIMJM97** [naverz/zepeto-studio-kor#1150] [5. Q&A]
안녕하세요 제페토를 시작한지 얼마 안 된 뉴비입니다..
빌드잇에서 3D 작업을 한 후, 제페토 스튜디오 가이드를 따라 유니티로 불러왔습니다.
ZepetoPlayers와 CharacterController를 통해서 월드에 제페토 캐릭터를 불러온 후 Play 버튼을 눌렀는데,
캐릭터가 바닥, 물체를 통과하여 계속 추락하기만 합니다.

해결방안이 없을까요?

**Comment by ZdevMonZ** [comment]
빌드잇에서 가져온 오브젝트에는 콜라이더가 없어서 직접 추가하셔야 해요.
